modules/about-dynamic-plug-ins.adoc

```diff
@@ -6,29 +6,41 @@
 [id="about-dynamic-plugins_{context}"]
 = About dynamic plug-ins
 
-A dynamic plug-in allows you to add custom pages and other extensions to your interface at runtime. The `ConsolePlugin` custom resource registers plug-ins with the console, and a cluster administrator enables plug-ins in the `console-operator` configuration.
+A dynamic plugin allows you to add custom pages and other extensions to your interface at runtime. The `ConsolePlugin` custom resource registers plugins with the console, and a cluster administrator enables plugins in the `console-operator` configuration.
 
 [id="dynamic-plugins-features"]
 == Key features
 
-A dynamic plug-in allows you to make the following customizations to the {product-title} experience:
+A dynamic plugin allows you to make the following customizations to the {product-title} experience:
 
 * Add custom pages.
-* Add perspectives and update navigation items.
+* Add perspectives beyond administrator and developer.
+* Add navigation items.
 * Add tabs and actions to resource pages.
 
-== PatternFly 4 guidelines
-When creating your plug-in, follow these guidelines for using PatternFly:
-
-* Use link:https://www.patternfly.org/v4/[PatternFly4] components and PatternFly CSS variables. Core PatternFly components are available through the SDK. Using PatternFly components and variables will help your plug-in look consistent in future console versions.
-* Make your plug-in accessible by following link:https://www.patternfly.org/v4/accessibility/accessibility-fundamentals/[PatternFly's accessibility fundamentals].
-* _Do not_ use other CSS libraries such as Bootstrap or Tailwind. They can conflict with PatternFly and will not match the console look and feel.
-
 [id="general-plug-in-guidelines"]
 == General guidelines
-When creating your plug-in, follow these general guidelines:
+When creating your plugin, follow these general guidelines:
 
-* Prefix your CSS class names with your plug-in name to avoid collisions. For example, `my-plugin_\_heading` and `my-plugin_\_icon`.
+* link:https://nodejs.org/en/[`Node.js`] and link:https://yarnpkg.com/[`yarn`] are required to build and run your plugin.
+* Prefix your CSS class names with your plugin name to avoid collisions. For example, `my-plugin_\_heading` and `my-plugin_\_icon`.
 * Maintain a consistent look, feel, and behavior with other console pages.
-* Use link:https://github.com/openshift/enhancements/blob/master/enhancements/console/dynamic-plugins.md#localization[react-i18next] for localization.
-* _Do not_ use console CSS classes in your markup or override console CSS classes. These are not APIs and are subject to change. Using them might break your plug-in. Avoid selectors like element selectors that could affect markup outside of your plug-in’s components.
+* Follow link:https://www.i18next.com/[react-i18next] localization guidelines when creating your plugin. You can use the `useTranslation` hook like the one in the following example:
++
+[source,ymal]
```

Comment:
```suggestion
[source,yaml]
```

Reply:
Although this snipped does not contain any yaml, just code snippet

Reply:
I changed to `[source,text]` to make more generic